cmake/DEPRECATED_osxbundle/Info.plist

```diff
@@ -3,17 +3,19 @@
 <plist version="1.0">
 <dict>
 	<key>CFBundleIconFile</key>
-	<string>AppIcon.icns</string>
+	<string>OBS.icns</string>
```

Comment:
GitHub is showing this as a file move.  Since #4556 was merged, do we need to keep this?  See also the rest of the files in `cmake/DEPRECATED_osxbundle`.

Reply:
Right, with that one merged the whole dir can go as well.